### DISCUSSION TITLE
How about every two days or every three weeks etc.

### DISCUSSION THREAD

**Discussion by huaraco** [xrutayisire/react-js-cron#20] [Ideas]
How about every two days or every three weeks etc. is it support this feature at the moment?

**Comment by xrutayisire** [comment]
Hello 😃 

All every can be done with "*/n", for example every 2 minutes "*/2 * * * * "

The problem with every two days in Cron syntax is that depending on the month it will not run as you want it.

For example:

<img width="384" alt="Screenshot 2022-03-24 at 17 11 02" src="https://user-images.githubusercontent.com/19946868/159972420-a34565a7-f2aa-4c09-8c95-b586d9f9cfec.png">

So if you really want to do every two days all over the years, you cannot use only the Cron syntax. Or at least not that I know. You will need some other management. But if "*/2" is good for you and you don't care if not really perfect use this.

For every 3 weeks  it's not possible only with the Cron syntax.

Anyway, react-js-cron is based on the Cron syntax, so if you look what you can do with the Cron syntax you will know what to do.
